exercises/food-chain/food_chain_test.py

```diff
@@ -1,108 +1,107 @@
 import unittest
 
-from food_chain import chain
-
-# Tests adapted from `problem-specifications//canonical-data.json` @ v1.0.0
-
-song = """I know an old lady who swallowed a fly.
-I don't know why she swallowed the fly. Perhaps she'll die.
-
-I know an old lady who swallowed a spider.
-It wriggled and jiggled and tickled inside her.
-She swallowed the spider to catch the fly.
-I don't know why she swallowed the fly. Perhaps she'll die.
-
-I know an old lady who swallowed a bird.
-How absurd to swallow a bird!
-She swallowed the bird to catch the spider that wriggled
-and jiggled and tickled inside her.
-She swallowed the spider to catch the fly.
-I don't know why she swallowed the fly. Perhaps she'll die.
-
-I know an old lady who swallowed a cat.
-Imagine that, to swallow a cat!
-She swallowed the cat to catch the bird.
-She swallowed the bird to catch the spider that wriggled
-and jiggled and tickled inside her.
-She swallowed the spider to catch the fly.
-I don't know why she swallowed the fly. Perhaps she'll die.
-
-I know an old lady who swallowed a dog.
-What a hog, to swallow a dog!
-She swallowed the dog to catch the cat.
-She swallowed the cat to catch the bird.
-She swallowed the bird to catch the spider that wriggled
-and jiggled and tickled inside her.
-She swallowed the spider to catch the fly.
-I don't know why she swallowed the fly. Perhaps she'll die.
-
-I know an old lady who swallowed a goat.
-Just opened her throat and swallowed a goat!
-She swallowed the goat to catch the dog.
-She swallowed the dog to catch the cat.
-She swallowed the cat to catch the bird.
-She swallowed the bird to catch the spider that wriggled
-and jiggled and tickled inside her.
-She swallowed the spider to catch the fly.
-I don't know why she swallowed the fly. Perhaps she'll die.
-
-I know an old lady who swallowed a cow.
-I don't know how she swallowed a cow!
-She swallowed the cow to catch the goat.
-She swallowed the goat to catch the dog.
-She swallowed the dog to catch the cat.
-She swallowed the cat to catch the bird.
-She swallowed the bird to catch the spider that wriggled
-and jiggled and tickled inside her.
-She swallowed the spider to catch the fly.
-I don't know why she swallowed the fly. Perhaps she'll die.
-
-I know an old lady who swallowed a horse.
-She's dead, of course!"""
-
-
-def verses(letter):
-    return letter.replace('die.', 'die.slice').split('slice')
-
-
-original = [verse.replace("\n", "").replace(" ", "").lower()
-            for verse in verses(song)]
-
-generated = [verse.replace("\n", "").replace(" ", "").lower()
-             for verse in verses(chain())]
+from food_chain import recite
 
 
+# Tests adapted from `problem-specifications//canonical-data.json` @ v2.1.0
+
 class FoodChainTest(unittest.TestCase):
 
     def test_fly(self):
-        self.assertEqual(original[0], generated[0])
+        expected = [
+            "I know an old lady who swallowed a fly."
```

Comment:
I think within each verse, we ought to keep the newlines (`\n`).

Reply:
But we seem to have reached a consensus in [beer-song](https://github.com/exercism/python/blob/master/exercises/beer-song/beer_song_test.py), [house](https://github.com/exercism/python/blob/master/exercises/house/house_test.py) and [twelve-days](https://github.com/exercism/python/blob/master/exercises/twelve-days/twelve_days_test.py), where there is no newline at the end of each verse.

Reply:
Hmm... I obviously didn't pay as much attention to `beer-song` and `house` as I thought. In the case of `twelve-days`, each verse is a single line, is this was not a relevant point.

After considering the format used in the canonical data, I have changed my mind and agree that it should remain this way to be consistent both with the other exercises and with the canonical data. However, this means that `house` is not actually consistent after all, as each verse contains multiple element in the returned list. This should be corrected.